### ISSUE TITLE
&:nth-child(#{$nth}+1) Fatal Error

### ISSUE THREAD

**Issue by iamhexcoder** [sasstools/sass-lint#1092]
Running version 1.10.2 through `gulp-sass-lint` version 1.3.2

---

I have the following in a .scss file:

```
@mixin some-mixin($nth: 1n) {
  
  &:nth-child(#{$nth}+1) {
    // stuff
  }
}
```

This causes a fatal error: `error  Please check validity of the block starting from line`. This occurs starting at the line containing `&:nth-child(#{$nth}+1) {`.

This outputs valid css, so I did not expect this fatal error. It also crashes gulp tasks if `.failOnError()` is optioned. I attempted to use the inline disable methods [here](https://github.com/sasstools/sass-lint#disabling-linters-via-source), as well as whole file, and via `ingnore` in the `.sass-lint.yml` file, but they do not do anything.

Is there anything that I am missing here?

**Comment by DanPurdy** [comment]
It's an issue with our AST unfortunately. There is nothing you can do to ignore it or skip it without ignoring the file as a whole or rewriting the Sass as it stands. Will have another look at this once Gonzales version 4 is updated.

**Comment by bgriffith** [comment]
This is fixed in the newly released version v1.11.0. Please let me know if you still have issues. Thanks!

**Comment by iamhexcoder** [comment]
Thanks, works like a charm!
